tutorial/README.md

```diff
@@ -0,0 +1,281 @@
+>
+    <ts:name>
+        <ts:string xml:lang="en">Coin</ts:string>
+    </ts:name>
+    <ts:contract name="Coin" interface="erc20">
+        <ts:address network="3">YOUR_CONTRACT_ADDRESS</ts:address>
+    </ts:contract>
+    <ts:origins>
+        <ts:ethereum contract="Coin"></ts:ethereum>
+    </ts:origins>
+    <ts:cards>
+        <!-- action cards will go here -->
+    </ts:cards>
+    <ts:attribute-types>
+        <!-- attributes will go here -->
+    <ts:attribute-types>
+</ts:token>
+```
+
+In the contract address, where the network is 3 for Ropsten, paste your contract address in place of `YOUR_CONTRACT_ADDRESS`.
+
+The UI elements rendering in studio (app.html) will be captured differently for TokenScript. Rather than a single view, individual actions are isolated in action cards.
+
+* For the action of Minting, create a new file called mint.shtml, and initialise with this template:
+```
+<script type="text/javascript"><![CDATA[
+class Token {
+    constructor(tokenInstance) {
+        this.props = tokenInstance
+
+    }
+
+    render() {
+        return `
+        <div class="ui container">
+            <div class="ui segment">
+
+            </div>
+        </div>`;
+    }
+}
+
+web3.tokens.dataChanged = (oldTokens, updatedTokens) => {
+    const currentTokenInstance = web3.tokens.data.currentInstance;
+    document.getElementById('root').innerHTML = new Token(currentTokenInstance).render();
+};
+
+]]></script>
+<div id="root"></div>
+
+```
+
+* mint.shtml: Inside the "ui segment" div, copy/paste contents of the "create" div from app.html
+  * notice the "create-address" and "create-address" input fields
+
+
+* Coin.xml: At the top of the xml (after the version tag), include references to front-end files `app.css` and `mint.shtml`:
+  * (For reference, full TokenScript file provided -> `tutorial-files/Coin.xml`)
+```
+<!DOCTYPE token  [
+    <!ENTITY style SYSTEM "app.css">
+    <!ENTITY mint.en SYSTEM "mint.shtml">
+    ]>
+```
+
+  * Coin.xml: Inside the <ts:cards> section, add the following <ts:action>. The english name is "Mint", and front-end files referenced (style and view).
+```
+        <ts:action>
+            <ts:name>
+                <ts:string xml:lang="en">Mint</ts:string>
+            </ts:name>
+            ...
+            <!-- attributetype tags -->
+            ...
+            <!-- transaction tag -->
+            ...
+            <style type="text/css">&style;</style>
+            <ts:view>&mint.en;</ts:view>
+        </ts:action>
+```
+
+  * Coin.xml: To refer to the input types that will be used in a smart contract call, add their attribute-type (notice "create-address" and "create-amount")
+
+```
+			...
+            <ts:attribute-type id="create-address" syntax="1.3.6.1.4.1.1466.115.121.1.15">
+                <ts:name>
+                    <ts:string xml:lang="en">Address to create</ts:string>
+                </ts:name>
+                <ts:origins>
+                    <!-- e18 is a hard coded multiplier.
```

Comment:
comment does not match the defined variable

Reply:
Had updated in file, now updated here.